pkg/component/etcd/etcd/bootstrap.go

```diff
@@ -258,6 +290,138 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 						Port:       metricsPort,
 						TargetPort: intstr.FromInt32(metricsPort),
 					},
+					{
+						Name:       webhookServerPortName,
+						Protocol:   corev1.ProtocolTCP,
+						Port:       webhookServerPort,
+						TargetPort: intstr.FromInt32(webhookServerPort),
+					},
+				},
+			},
+		}
+
+		failurePolicy     = admissionregistrationv1.Fail
+		matchPolicy       = admissionregistrationv1.Exact
+		sideEffects       = admissionregistrationv1.SideEffectClassNone
+		scope             = admissionregistrationv1.AllScopes
```

Comment:
Remove these variables. You can use `ptr.To()` instead.

Reply:
Done